eng/targets/RepoToolset/Compiler.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- All Rights Reserved. Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
```

Comment:
If/when dev16 is a requirement, you can stop adding this.  MSBuild now handles it for you, and adding it just leads to annoying LOH allocations.  

cc @rainersigwald @drewnoakes

Reply:
yup.